js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +287,41 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        // the ships from data doesn't exist yet
+        // shipsFromCountry: this.model.get('listing').get('shipsFrom');
+        countryData: this.countryData,
+        defaultCountry: this.defaultCountry,
       }));
 
       super.render();
 
       this._$deleteListing = null;
       this._$shipsFreeBanner = null;
       this._$popInMessages = null;
+      this._$photoSection = null;
+      this._$shippingOptions = null;
+      this._$photoRadioBtns = null;
+
+      // commented out until variants are available
+      // this.$('.js-variantSelect').select2();
+
+      const initialPhotoHash = this.model.get('listing').toJSON().item.images[0].original;
+      const initialPhoto = app.getServerUrl(`ipfs/${initialPhotoHash}`);
+
+      setTimeout(() => {
+        this.$photoSection.cropit({
+          smallImage: 'stretch',
+          allowDragNDrop: false,
+        });
+
+        this.$photoSection.cropit('imageSrc', initialPhoto);
+      }, 0);
+
+      this.$('#shippingDestinations').select2({
+        placeholder: app.polyglot.t('listingDetail.shipToPlaceholder'),
```

Comment:
Why is this placeholder necessary? Will there ever be a scenario when it's shown?